approvaltests/reporters/python_native_reporter.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import filecmp
+import os
+import sys
+from difflib import unified_diff
+
+from approvaltests.reporters import get_command_text
+
+
+def calculate_diff(file1, file2):
+    if filecmp.cmp(file1, file2):
+        return "Files are identical"
+    else:
+        with open(file1) as f1:
```

Comment:
If anyone experiences newline problems related to approval files created on different systems we my want to include `'U'` in the mode here. See https://docs.python.org/2/library/functions.html#open